bot/commands/utility/listScheduled.js

```diff
@@ -0,0 +1,43 @@
+const source = require('rfr');
+// const moment = require('moment');
+
+const logger = source('bot/utils/logger');
+const { sendMessage } = source('bot/utils/util');
+const agenda = source('scheduler');
+
+function scheduleMessage(message) {
+    agenda.jobs({
+        'data.serverID': message.guild.id,
+    })
+        .then((jobs) => {
+            logger.info(jobs);
+            sendMessage(message.channel, 'there are a bunch of jobs');
+        });
+}
+
+module.exports = {
+    args: 0,
+    name: 'scheduled-jobs',
+    botAdmin: false,
+    description: 'Schedule a recuring message to be sent to the channel that this is called from',
+    usage: '<day of the week> at <time>\n<some message to be sent>',
+    aliases: [],
+    execute: scheduleMessage,
+    docs: `#### Schedule Message
```

Comment:
I think you forgot to update docs field?

Reply:
fixed in 3b66bcf41c99fa3390cf7d0c2bca01618b295549